app/Directory.Packages.props

```diff
@@ -43,5 +43,8 @@
     <PackageVersion Include="System.Linq.Async" Version="6.0.1" />
     <PackageVersion Include="xunit.runner.visualstudio" Version="2.5.4" />
     <PackageVersion Include="xunit" Version="2.6.2" />
+    <PackageVersion Include="ApprovalTests" Version="5.2.4" />
```

Comment:
Consider using something else, this project is no longer maintained. https://github.com/approvals/ApprovalTests.Net?tab=readme-ov-file#-approvaltests